src/test/java/org/cactoos/io/LoggingInputStreamTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.io;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.hamcrest.core.IsNull;
+import org.junit.Test;
+
+/**
+ * Test case for {@link LoggingInputStream}.
+ *
+ * @since 0.39
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class LoggingInputStreamTest {
+
+    @Test(expected = IOException.class)
```

Comment:
@ilyakharlamov Use `Throws` instead.

Reply:
I tried `find cactoos/src -type f -name "*.java" |xargs grep Throws` - nothing. Not sure what syntax is that for Throws? Usage example?

Reply:
waiting for #1023 I guess

Reply:
@ilyakharlamov #1023 has been closed. Go ahead!

Reply:
@fabriciofx How is that, can you give me an example `org.llorllale.cactoos.matchers.Throws` method `matchesSafely` is package-private so I can't use that, WDYT?

Reply:
Let's wait for [cactoos-matchers#90](https://github.com/llorllale/cactoos-matchers/issues/90).

Reply:
@ilyakharlamov Ok. But I thought better and won't wait for cactoos-matchers. Let's go ahead.